src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';').map(style =>
+    style.split(':').map(styleRule => styleRule.trim()));
```

Comment:
```suggestion
  const stylesArray = sourceString
    .split(';')
    .map(style => style.split(':')
      .map(styleRule => styleRule.trim()));
```

Reply:
agree, each method should be on the new line